### PR TITLE
Homework_Day_2

### PR DESCRIPTION
**Pull request by dmlatesting**

### CODE REVIEW

File: Day-2-Exercises/index.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+//Exercise: Level 2
+
+//Using console.log() print out the following statement:
+console.log('The quote \'There is no exercise better for the heart than reaching down and lifting people up.\' by John Holmes teaches us to help one another.')
+console.log('/"Love is not patronizing and charity isn\'t about pity, it is about love. Charity and love are the same -- with charity you give love, so don\'t just give money but reach out your hand instead.\"')
+
+//Check if typeof '10' is exactly equal to 10. If not make it exactly equal.
+console.log(typeof(parseInt('10')) == typeof(10))
+console.log(typeof('10') == typeof(toString(10)))
```
Comment (jkalandarov) [inline Day-2-Exercises/index.js]: There are two equality operators in JS: [normal ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality)(==) and [strict ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)(===). 
This task is about these operators. `typeof` operator always returns string value type.

File: Day-2-Exercises/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+let a = 'python '
+let b = 'jargon'
+console.log(a.includes('on') && b.includes('on') )
+
+//I hope this course is not full of jargon. Check if jargon is in the sentence.
+let sentence = 'I hope this course is not full of jargon.'
+console.log(sentence.includes('jargon'))
+
+//Generate a random number between 0 and 100 inclusively.
+console.log(Math.floor(Math.random()*100))
```
Comment (jkalandarov) [inline Day-2-Exercises/index.js]: You solution generates random number between 0 and 99. You should include 100, too.

File: Day-2-Exercises/index.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+//Generate a random number between 50 and 100 inclusively.
+console.log(50 + Math.floor(Math.random()*50))
```
Comment (jkalandarov) [inline Day-2-Exercises/index.js]: generates a random number between 50 and 99.

File: Day-2-Exercises/index.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+//Generate a random number between 0 and 255 inclusively.
+console.log(Math.floor(Math.random()*255))
```
Comment (jkalandarov) [inline Day-2-Exercises/index.js]: generated a random number between 0 and 254

File: Day-2-Exercises/index.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+console.log(badSentence.replace(cleanPattern,''))
+
+
+
+//Calculate the total annual income of the person by extracting the numbers from the following text. 'He earns 5000 euro from salary per month, 10000 euro annual bonus, 15000 euro online courses per month.'
+let incomeText = 'He earns 5000 euro from salary per month, 10000 euro annual bonus, 15000 euro online courses per month.'
+let result = 0
+let incomeArray = incomeText.split(" ")
+console.log('Annual income: ', parseInt(incomeArray[2]) * 12 + parseInt(incomeArray[8]) + parseInt(incomeArray[12]) * 12)
```
Comment (jkalandarov) [inline Day-2-Exercises/index.js:102]: Good job. Can you do it using RegExp (regular expression) to extract numbers from the text?

**Review by jkalandarov — CHANGES_REQUESTED** [review]

File: Day-2-Exercises/index.js (hunk 1 above)
Comment (dmlatesting) [reply Day-2-Exercises/index.js]: Updated in the code

File: Day-2-Exercises/index.js (hunk 2 above)
Comment (dmlatesting) [reply Day-2-Exercises/index.js]: Updated in the code

File: Day-2-Exercises/index.js (hunk 3 above)
Comment (dmlatesting) [reply Day-2-Exercises/index.js]: Updated in the code

File: Day-2-Exercises/index.js (hunk 4 above)
Comment (dmlatesting) [reply Day-2-Exercises/index.js]: Updated in the code

File: Day-2-Exercises/index.js (hunk 5 above)
Comment (dmlatesting) [reply Day-2-Exercises/index.js:102]: Added as second option to calculate

**Review by jkalandarov — APPROVED** [review]

**Pull request merged**